CONTRIBUTING.md

```diff
@@ -20,4 +20,32 @@ reason about the proposed changes.
 Many of the tests require privileges to set resource limits and load eBPF code.
 The easiest way to obtain these is to run the tests with `sudo`:
 
-    sudo go test ./...
\ No newline at end of file
+    sudo go test ./...
+
+## Other platforms
+
+The main development platform that the project targets is `amd64`. However running
+the code and the examples should not be hard on any platform supported by `go` and
+`docker`. Here are some hints for `arm64`, adding other platforms should be similar.
+The suggested approach will always use `amd64` build tools, but allow to cross-build
+and use them with an emulation.
+
+#### Ubuntu `linux/arm64`
```

Comment:
So Linux == Ubuntu? :) 

If I understand correctly, this section is supposed to describe how to generate a Docker image containing an arm64-based LLVM toolchain. Can't tell for sure because there's no conclusion to the description. This would enable developers running arm64-based Linux distributions as a development environment (which might include Docker-on-M1), is that correct?

If so, I would remove all of the fluff here and make `testdata/docker` rely on buildx. A short mention can be included in the readme there about `qemu-user-static` and `binfmt-support` being required, although this is kind of expected for running a cross-compilation toolchain. `buildx` will likely fail and tell the user these tools are required.